### PR TITLE
Milestone 2: Work section

### PR DESCRIPTION
**Pull request by kimitaWanjohi**
- Added work section and project cards
- Added About section
- Added Languages, Skills, and Frameworks cards.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,12 +56,221 @@ <h2>I'm a software developer</h2>
...
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="row">
+                    
+                    <div class="about-card">
+                        <div>
+                            <img src="assets/icon-frameworks.svg" alt="">
+                        </div>
```
Comment (Lizdev-05) [inline index.html:247]: Great job so far 👍. Kindly center your about-me card so it matches the Figma design, it's currently not well aligned 😊 ☺️ 

image

File: style.css [hunk 2]
```diff
@@ -53,24 +66,197 @@ body {
...
+  margin-top: 1rem;
+}
+
+.row {
+  margin-top: 2rem;
+  display: grid;
+  justify-content: center;
+  width: 100%;
+}
+
```
Comment (Lizdev-05) [inline style.css:129]: Great work on the styling, it is perfect 👍 . But according to the requirement, the work main container should be displayed grid, kindly adjust your styling to use a grid display. 😉 😊 💪

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @kimitaWanjohi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct GitHub Flow 👍

- Good commit history 💰

- Good PR title/description 💯

- All linters passed 🏅

- Personalised README file 👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (kimitaWanjohi) [reply style.css:129]: @Lizdev-05  I am using Grid already. I have named it row so I can reuse it in the Languages, Frameworks and Skill.

File: index.html (hunk 1 above)
Comment (kimitaWanjohi) [reply index.html:247]: I have centered the about cards

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @kimitaWanjohi :wave: 
 
Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: Readme is professional.
:heavy_check_mark: Descriptive pull request.
:heavy_check_mark: Meaningful commit message.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**